R/PipeOpLearner.R

```diff
@@ -118,6 +119,13 @@ PipeOpLearner = R6Class("PipeOpLearner", inherit = PipeOp,
         }
       }
       private$.learner
+    },
+    predict_type = function(val) {
+      if (!missing(val)) {
+        assert_subset(val, names(mlr_reflections$learner_predict_types[[private$.learner$task_type]]))
```

Comment:
doesn't the learner itself also assert this?

Reply:
pipeops can / could have AB's that are not learner's. But those might assert this themselves aswell.

Reply:
we should think about what PipeOpThreshold does in this case. There we can not set to `prob`.
this should then possibly error (within PipeOpThreshold I guess).